unison-src/transcripts/tdnr.output.md

```diff
@@ -0,0 +1,1008 @@
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
+    
+      bad.foo : Text
+
+```
+TDNR selects local term (in namespace) that typechecks over local term (in file) that doesn't.
+
+``` unison
+good.foo = 17
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      good.foo : Nat
+
+```
+``` ucm
+scratch/main> add
+
+  ⍟ I've added these definitions:
+  
+    good.foo : Nat
+
+```
+``` unison
+bad.foo = "bar"
+thing = foo Nat.+ foo
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      bad.foo : Text
+      thing   : Nat
+
+```
+TDNR selects local term (in namespace) that typechecks over local term (in namespace) that doesn't.
+
+``` unison
+good.foo = 17
+bad.foo = "bar"
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      bad.foo  : Text
+      good.foo : Nat
+
+```
+``` ucm
+scratch/main> add
+
+  ⍟ I've added these definitions:
+  
+    bad.foo  : Text
+    good.foo : Nat
+
+```
+``` unison
+thing = foo Nat.+ foo
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      thing : Nat
+
+```
+TDNR selects local term (in namespace) that typechecks over local term (shadowing namespace) that doesn't.
+
+``` unison
+good.foo = 17
+bad.foo = "bar"
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      bad.foo  : Text
+      good.foo : Nat
+
+```
+``` ucm
+scratch/main> add
+
+  ⍟ I've added these definitions:
+  
+    bad.foo  : Text
+    good.foo : Nat
+
+```
+``` unison
+bad.foo = "baz"
+thing = foo Nat.+ foo
+```
+
+``` ucm
+
+  Loading changes detected in scratch.u.
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      thing : Nat
+    
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
+    
+      bad.foo : Text
+
+```
+TDNR selects local term (shadowing namespace) that typechecks over local term (in file) that doesn't.
```

Comment:
On the examples that select the shadowing namespace, do we need to show the result to know that it's selecting in the shadowing version?